### ISSUE TITLE
Composer versioning for oe_theme (locked at 0.9.0)

### ISSUE THREAD

**Issue by vever001** [openeuropa/drupal-site-template#26]
On a fresh install I noticed the version of oe_theme is 0.9.0, while the latest is 0.13.0.
Is this intentional?
If not we might need to change `"openeuropa/oe_theme": "^0.9"` (meaning >=0.9.0 <0.10.0) to `"openeuropa/oe_theme": "~0.13"`

This is also probably the case for openeuropa/oe_profile using a carret and pre-1.0 version.
https://getcomposer.org/doc/articles/versions.md#caret-version-range-

**Comment by voidtek** [comment]
This issue seems to be resolved.

During the `composer install`
```
  - Installing openeuropa/oe_theme with artifact from https://github.com/openeuropa/oe_theme/releases/download/1.5.0/oe_theme-1.5.0.tar.gz.
  - Installing openeuropa/oe_theme (1.5.0): Downloading (100%)  
```

**Comment by vever001** [comment]
Thx @voidtek 
Yes, I confirm, it was resolved on May 15 by the 1.0 release.
https://github.com/openeuropa/drupal-site-template/commit/c32dc5493b62190d66b9c299e80e2a84a6430d01#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780
